tests/ImageSharp.Tests/Formats/Jpg/ReferenceImplementations.cs

```diff
@@ -56,9 +362,11 @@ private static void iDCT1Dllm_32f(MutableSpan<float> y, MutableSpan<float> x)
             float a0, a1, a2, a3, b0, b1, b2, b3;
             float z0, z1, z2, z3, z4;
 
+            float r0 = 1.414214f;
```

Comment:
I intentionally wanted to keep it similar to the original C++ code. Even it's mistakes can serve as documentation if someone ever goes into DCT implementation details. This is not the core codebase, it's here for testing & documentation.